test-project/gradlew

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
```

Comment:
is this intentionally committed?

Reply:
Yes, I added a commit with this, as it helps to run locally any test-project.